src/library/Uno.Themes.WinUI.Markup/Theme/ProgressBar.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class ProgressBar
+		{
+			public static class Resources
+			{
+				[ResourceKeyDefinition(typeof(Brush), "ProgressBarForeground")]
+				public static ResourceValue<Brush> Foreground => new("ProgressBarForeground", true);
```

Comment:
Just needs to be updated with latest master and then replace ResourceValue with `ThemeResourceKey`